test/generator/stubuser_generator.cpp

```diff
@@ -54,6 +54,7 @@ class StubUser : public Halide::Generator<StubUser> {
         gp.untyped_buffer_output_type = int32_buffer_output.type();
         gp.intermediate_level.set(LoopLevel(calculated_output, Var("y")));
         gp.vectorize = true;
+        gp.str_param = "2 * x y +";
```

Comment:
Nit: this particular RPN expression would underflow the stack.

Reply:
Ops!